spec/test/stored_queries/test_fulltext_search_flags.py

```diff
@@ -0,0 +1,287 @@
+"""
+Tests the boolean flags on NCBI Taxon nodes when used in conjunction with full text search, e.g.
+in the queries:
+- taxonomy_search_species_strain
+- taxonomy_search_species_strain_no_sort
+
+Note these tests omit several fields from the records that are not relevant to the queries.
+"""
+
+import os
+import yaml
+from arango import ArangoClient
+from frozendict import frozendict  # type: ignore[attr-defined]
+from pathlib import Path
+from pytest import fixture
+from relation_engine_server.utils.config import get_config
+from relation_engine_client import REClient
+
+_QUERY_SORT = "taxonomy_search_species_strain"
+_QUERY_NO_SORT = "taxonomy_search_species_strain_no_sort"
+
+
+@fixture(scope="module")
+def conf():
+    config = get_config()
+    config["re_api_url"] = os.environ.get("RE_API_URL", "http://localhost:5000")
```

Comment:
Wait why is this not in the config already ... :googlymem:

Reply:
No idea. This is how it's done in other tests so I just copied it

Reply:
Oh wait, it's probably not in the RE API config because the API doesn't need to know it's own URL.

Reply:
Oh ...